version/version.go

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package version
+
+var (
+	// Version information for the Solr Operator
+	Version       string = "v0.3.0"
+	VersionSuffix string = "prerelease"
+	BuildTime     string
```

Comment:
We don't want this because it means that builds are not reproducible. There may be other reasons that the build is not reproducible, but this we shouldn't add to it if we can help it.

Reply:
Maybe we bake this in during a release, therefore it would be reproducible? Rename it "Release Date" or something

Reply:
I'm actually kind of lukewarm on this, the only binary releases we are doing are docker images, which have questionable reproducibility anyways.